snappiershot/snapshot/metadata.py

```diff
@@ -1,11 +1,48 @@
 """ Metadata object used to identity and track snapshots. """
-from typing import Dict
+from typing import Any, Dict
+
+from numpy import all, ndarray
 
 from ..inspection import CallerInfo
 
 
+def compare_metadata(from_test_function: Any, from_snapshot: Any) -> bool:
+    """Instantiate object from a dictionary read from file in order to compare to SnappierShot input.
+
+    SnappierShot serializes class objects by decomposing them into their parts in the form of a dictionary, then
+    writes to the snapshot JSON. Thus, SnappierShot must attempt to re-instantiate the dictionary form of the
+    objects when comparing inputs to a test function to those inputs serialized to JSON.
+
+    Args:
+        from_test_function: Inputs to the test function calling SnappierShot
+        from_snapshot: Inputs to the test function after being serialized to JSON
+    """
+    result = False
+
+    # Determine if the metadata can instantiate a class from a dictionary, or whether any object is None
+    metadata_function_has_method = hasattr(from_test_function, "from_dict")
+    metadata_function_is_not_none = from_test_function is not None
```

Comment:
suggest to just move these two `is not None` statements directly into the elif call since they're not used multiple times.

Reply:
fixed [3d6c365](https://github.com/MORSECorp/snappiershot/pull/96/commits/3d6c36553e50807a3b951ca3bf4ffd7c8cfaa97f)